src/Fixer/ClassNotation/NoUnneededFinalMethodFixer.php

```diff
@@ -120,7 +120,7 @@ private function fixClass(Tokens $tokens, $classOpenIndex, $classIsFinal)
      */
     private function isPrivateMethod(Tokens $tokens, $index, $classOpenIndex)
     {
-        $index = max($classOpenIndex + 1, $tokens->getPrevTokenOfKind($index, ['{', '}']));
+        $index = max($classOpenIndex + 1, $tokens->getPrevTokenOfKind($index, [';', '{', '}']));
```

Comment:
Apologies for the problems, @Slamdunk, it was my mistake here not to think of examples where this can go wrong.

Reply:
No need to apologize, as the creator of this Fixer I reviewed your code and didn't notice it until I applied the fixer to the codebase

Reply:
Great fixer, by the way!